doc/releases/changelog-0.41.0.md

```diff
@@ -398,75 +398,99 @@ capture enabled by adding :func:`qml.capture.enable() <pennylane.capture.enable>
+  @qml.qnode(dev)
+  def circuit(mat):
+      qml.QubitUnitary(sparse_mat, wires=range(13))
+      return qml.state()
+  ```
 
+* Support for sparse representation in operators is expanded.
+  
+  Sparse input is now possible for:
+  * `qml.StatePrep` [(#6863)](https://github.com/PennyLaneAI/pennylane/pull/6863)
+  * `qml.QubitUnitary` [(#6889)](https://github.com/PennyLaneAI/pennylane/pull/6889) [(#6986)](https://github.com/PennyLaneAI/pennylane/pull/6986) [(#7143)](https://github.com/PennyLaneAI/pennylane/pull/7143)
+  * `qml.BlockEncode` [(#6963)](https://github.com/PennyLaneAI/pennylane/pull/6963) [(#7140)](https://github.com/PennyLaneAI/pennylane/pull/7140)
+  ```python
+    import scipy
+
+    sparse_state = scipy.sparse.csr_array([0,1,0,0])
+    sparse_mat = scipy.sparse.csr_array(qml.X.compute_matrix())
+
+    dev = qml.device('default.qubit')
+    @qml.qnode(dev)
+    def circuit():
+        qml.StatePrep(sparse_state, wires=range(2))
+        # qml.BlockEncode(sparse_mat,wires=range(2))
+        qml.QubitUnitary(sparse_mat, wires=range(1))
+        qml.ctrl(qml.QubitUnitary(sparse_mat,wires=0),control=1)
+        return qml.state()
+    ```
+    ```pycon
+    >>> circuit()
+    array([0.+0.j, 1.+0.j, 0.+0.j, 0.+0.j])
+    ```
+  
+  Sparse representations can be obtained for:
+  * `qml.SWAP` [(#6965)](https://github.com/PennyLaneAI/pennylane/pull/6965)
+  * `Controlled` operations [(#6994)](https://github.com/PennyLaneAI/pennylane/pull/6994)
+  
   ```pycon
-  >>> import numpy as np
-  >>> import pennylane as qml
-  >>> import scipy as sp
-  >>> U_dense = np.eye(4)  # 2-wire identity
-  >>> U_sparse = sp.sparse.csr_matrix(U_dense)
-  >>> op = qml.QubitUnitary(U_sparse, wires=[0, 1])
+  >>> op_matrix = qml.SWAP(wires=[0,1]).sparse_matrix()
+  >>> type(op_matrix)
+  <class 'scipy.sparse._coo.coo_matrix'>
+  >>> print(op_matrix)
+  (0, 0)	1
+  (1, 2)	1
+  (2, 1)	1
+  (3, 3)	1
+  >>> sparse_mat = scipy.sparse.csr_array(qml.X.compute_matrix())
+  >>> op = qml.ctrl(qml.QubitUnitary(sparse_mat,wires=0),control=1)
+  >>> type(op.sparse_matrix())
   >>> print(op.sparse_matrix())
-  <Compressed Sparse Row sparse matrix of dtype 'float64'
-          with 4 stored elements and shape (4, 4)>
-    Coords        Values
-    (0, 0)        1.0
-    (1, 1)        1.0
-    (2, 2)        1.0
-    (3, 3)        1.0
-  >>> op.sparse_matrix().toarray()
-  array([[1., 0., 0., 0.],
-        [0., 1., 0., 0.],
-        [0., 0., 1., 0.],
-        [0., 0., 0., 1.]])
+  (0, 0)	1
+  (1, 1)	1
+  (2, 3)	1
+  (3, 2)	1
   ```
 
-* `qml.StatePrep` now accepts sparse state vectors. Users can create `StatePrep` using `scipy.sparse.csr_matrix`. Note that non-zero `pad_with` is forbidden.
-  [(#6863)](https://github.com/PennyLaneAI/pennylane/pull/6863)
+* All operators that support `Operator.sparse_matrix` can now choose a format for the sparse matrix and support the wire_order parameter [(#6995)](https://github.com/PennyLaneAI/pennylane/pull/6995).
```

Comment:
```suggestion
* Operators that have a `sparse_matrix` method can now choose a sparse-matrix format and the order of their wires.
  [(#6995)](https://github.com/PennyLaneAI/pennylane/pull/6995).
```

Also might not hurt to mention what the formats are and what their keyword arg string is :)